### ISSUE TITLE
Adjust topics you're comfortable with

### ISSUE THREAD

**Issue by thomasve-DANS** [tvanerven/amy#10]
<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-3dd86553-7fff-783a-2301-a566bb9de8e6"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Adjust </span><span style="font-size:10.5pt;font-family:Roboto,sans-serif;color:#212529;background-color:#ffffff;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Topic and lessons you're comfortable teaching:</span></p><br /><br /><div dir="ltr" style="margin-left:0pt;" align="left">

  | SoRDS curriculum, please adjust
-- | --
dc/cloud | Software carpentry - command line
  | Software carpentry - git
  | Software carpentry - R
  | Software carpentry - Python
dc/python | Visualisation
dc/r | Open and Responsible Research
dc/shell | Research Data Management
dc/spreadsheets |  
dc/sql | Computational Infrastructures
swc/git | Information Security
swc/hg | Author Carpentry
swc/make | Machine Learning
swc/matlab | Neural Networks
swc/python |  
swc/r | Data stewardship
swc/regexp |  
swc/shell |  
swc/sql |  
swc/svn |  

</div></b>

**Comment by hughshanahan** [comment]
I"m a little confused with the first column - I assume that's the present set of entries?

**Comment by pherterich** [comment]
> I"m a little confused with the first column - I assume that's the present set of entries?

Yes, that's what's currently in the system and I don't even think that represents a lot of the carpentries. We basically need to figure how detailed the SoRDS curriculum should be reflected here.

**Comment by hughshanahan** [comment]
Is this sufficient now and can be closed or is there anything missing here?

**Comment by pherterich** [comment]
The SoRDS curriculum still needs to be implemented, if we confirm that the list in the right column is complete, that should be easy to do.

**Comment by thomasve-DANS** [comment]
I've gone ahead and added this to the codebase; please let me know if this list is complete before I deploy it.

**Comment by pherterich** [comment]
The list looks pretty complete to me, it would be great to see that implemented instead of the standard software carpentry stuff. I assume this is something that could be easily adjusted through the interface if needed?

**Comment by tvanerven** [comment]
Yeap, it can be. We discussed yesterday, I'll also add a few options to group/categorize them more effectively based on Hugh's input.
